test/gen/Makefile

```diff
@@ -1,8 +1,9 @@
 
+SHELL := /bin/bash
```

Comment:
why is this needed?  I use `sh`

Reply:
Without this the `make build` fails with this message:
```
/bin/sh: 1: source: not found
Makefile:15: recipe for target 'build-python' failed
make: *** [build-python] Error 127
```
I googled for "/bin/sh: 1: source: not found" and found this solution:
https://stackoverflow.com/questions/670191/getting-a-source-not-found-error-when-using-source-in-a-bash-script

I don'n know, what environment are you using for `sh` to find `source`?